common/src/commonMain/kotlin/io/xorum/codeforceswatcher/db/DatabaseQueries.kt

```diff
@@ -0,0 +1,70 @@
+package io.xorum.codeforceswatcher.db
+
+import io.xorum.codeforceswatcher.features.contests.models.Contest
+import io.xorum.codeforceswatcher.features.problems.models.Problem
+import io.xorum.codeforceswatcher.features.users.models.RatingChange
+import io.xorum.codeforceswatcher.features.users.models.User
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonConfiguration
+import kotlinx.serialization.list
+
+object DatabaseQueries {
```

Comment:
Nice idea with and syntax with `object`, but we will need to change it later, because `object` doesn't work on iOS.